### PR TITLE
Detect legacy relevancy during import as well

### PR DESCRIPTION
**Pull request by lukaszachy**
Respects priority of StructuredField same way as existing tooling

Fixes: #1271

### CODE REVIEW

**Review by hegerj — APPROVED** [review]
looks good

File: tests/integration/test_nitrate.py [hunk 1]
```diff
@@ -278,3 +278,26 @@ def test_basic(self):
...
+        runner = CliRunner()
+        self.runner_output = runner.invoke(
+            tmt.cli.main, [
+                "test", "import", "--nitrate"], catch_exceptions=False)
+        self.assertEqual(self.runner_output.exit_code, 0)
+
+        tree_f36_intel = tmt.Tree(
+            '.',
+            context={
+                'distro': ['fedora-36'],
```
Comment (psss) [inline tests/integration/test_nitrate.py:295]: What about trying also with `fedora-35` to exercise the second relevancy rule as well?

**Review by psss — APPROVED** [review]
Looks good, just one minor test suggestion.

File: tests/integration/test_nitrate.py (hunk 1 above)
Comment (lukaszachy) [reply tests/integration/test_nitrate.py:295]: Sure, hopefully requre stays happy.

**Pull request merged**